lib/v3/parser.js

```diff
@@ -90,6 +88,53 @@ class Parser extends EventEmitter {
         this.emit('end');
     }
 
+    static getDefaultOptions() {
+        return {
+            includeSourceLocations: true,
+            defaultMethodVisibility: utils.DEFAULT_VISIBILITY,
+            defaultActionVisibility: utils.DEFAULT_VISIBILITY,
+            features: [...SUPPORTED_FEATURES],
+            componentName: null,
+            eventsEmitted: {},
+            identifiers: {},
+            imports: {},
+            dispatcherConstructorNames: [],
+            dispatcherNames: [],
+        };
+    }
+
+    static normalizeOptions(options) {
+        const defaults = Parser.getDefaultOptions();
```

Comment:
Looks like that this method partially duplicated with V2 parser and can be extracted to common place